### PR TITLE
Add an open date option to open the datepicker on a certain date

### PR DESCRIPTION
**Pull request by pdong**
This addresses  #319 and #85 .

Let me know your thoughts.  Thanks!

### CODE REVIEW

**Comment by charliekassel** [comment]
Thanks @pdong - this feature has been asked for a few times.
Due to an earlier merge this now has conflicts - can you take a look please?

**Comment by pdong** [comment]
Sorry for not keeping up to date with this; I'll take a look at resolving the merge conflict tonight. 👍

**Comment by pdong** [comment]
@charliekassel should be able to merge now. Let me know if that works for you.

File: src/components/Datepicker.vue [hunk 1]
```diff
@@ -143,13 +149,14 @@ export default {
     dayViewOnly: Boolean
   },
   data () {
+    var startDate = this.openDate ? new Date(this.openDate) : new Date()
```
Comment (charliekassel) [inline src/components/Datepicker.vue]: lets make this a `const`

File: src/components/Datepicker.vue [hunk 2]
```diff
@@ -172,6 +179,9 @@ export default {
     value (value) {
       this.setValue(value)
     },
+    openDate () {
```
Comment (charliekassel) [inline src/components/Datepicker.vue]: Don't think we need this method?

File: src/components/Datepicker.vue [hunk 3]
```diff
@@ -740,7 +750,11 @@ export default {
     },
     setPageDate (date) {
       if (!date) {
-        date = new Date()
+        if (this.openDate) {
+          date = new Date(this.openDate)
```
Comment (charliekassel) [inline src/components/Datepicker.vue]: This line is missed in the tests.
https://coveralls.io/builds/14019344/source?filename=src%2Fcomponents%2FDatepicker.vue#L754

**Pull request merged**